spec/db/migrate/20210401171237_move_saml_client_cert_to_certs_spec.rb

```diff
@@ -1,35 +0,0 @@
-require 'rails_helper'
```

Comment:
**Question:** Is there a reason why you removed this spec file?

Reply:
Because it's useless if the `saml_client_cert` column doesn't exist anymore. Either we'd have to stub it (ew) or find a way to roll the DB back to a particular migration just for these tests

Reply:
🤦 that makes perfect sense, thanks!

Reply:
That's probably why you don't normally write migration tests 😛.